ts/DataGrid/DataGridOptions.d.ts

```diff
@@ -170,6 +175,13 @@ export interface CellValue {
     value: DataTable.CellType
 }
 
+/**
+ * Contains events for row
+ */
+export interface DataGridEvents {
+    row?: Record<string, Function>
```

Comment:
I would add an extra interface for the row events and IMO each event should have its own, public event callback interface.